### PR TITLE
aikido changes

### PR DESCRIPTION
**Pull request by pengjujin**
new urdf and tag json files to make pr_ordata compatible with aikido

### CODE REVIEW

**Comment by gilwoolee** [comment]
What are the updates here? Could you resolve the conflicts? Some of those URDFs in the master branch might have been recently added by @yskim041.

File: data/objects/plastic_bowl.kinbody.xml [hunk 1]
```diff
@@ -17,3 +17,4 @@
     </spheres>
   </orcdchomp>
 </KinBody>
+I've mos
```
Comment (pengjujin) [inline data/objects/plastic_bowl.kinbody.xml]: Remove this line

File: data/objects/table_mike.urdf [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      <geometry>
+        <cylinder radius="0.015" length="0.01"/>
+      </geometry>
+      <material name="dark_gray">
+        <color rgba="0.4 0.4 0.4 1.0"/>
+      </material>
+    </visual>
+  </link>
+</robot>
+Status API Training Shop Blog About
```
Comment (pengjujin) [inline data/objects/table_mike.urdf]: Remove this line

File: data/objects/aikido_tag_data_bowl_glass.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      ,
+      
+        0,
+        0,
+        0,
+        1
+      
+    ]
+  },
+}
```
Comment (pengjujin) [inline data/objects/aikido_tag_data_bowl_glass.json]: remove new line

**Review by pengjujin — COMMENTED** [review]
remove some of the random string in the urdf please

**Review by brianhou — CHANGES_REQUESTED** [review]
Not all the tags from `tag_data.json` are in `aikido_tag_data.json`, so it's not yet a drop-in replacement. It seems like it might just be a simple re-formatting from one to the other; if that's the case, we should do that before merging this PR.

Nit: can we rename `aikido_tag_data.json` to something more meaningful? Perhaps `urdf_tag_data.json`, if that's the main difference.

**Comment by jslee02** [comment]
Does anyone want to take over this PR? :smiley:

**Comment by youngsunkim** [comment]
I already added new tag data files that follow the theme of this PR in #32. I will look into this and clean up, or we can just drop this PR since we are not using the new objects in this PR anymore.

**Pull request closed without merging**